zap/src/main/java/org/parosproxy/paros/network/HttpHeader.java

```diff
@@ -47,6 +47,7 @@
 // ZAP: 2022/09/12 Allow only major HTTP version.
 // ZAP: 2022/11/17 Add HTTP/2 constant.
 // ZAP: 2022/11/22 Lower case the HTTP field names for compatibility with HTTP/2.
+// ZAP: 2023/08/15 Add FORM_MULTPART_CONTENT_TYPE.
```

Comment:
Should the comment and the variable (and it's use I guess) include the "I" in multipart? (I think so)

Reply:
Good catch.

Reply:
Ha!

Reply:
Fixed 😁